view_point_extractor/ViewPoint[] view_point_vec
